src/Webhooks/Topics.php

```diff
@@ -38,6 +38,7 @@ final class Topics
     public const CUSTOMERS_DISABLE = 'CUSTOMERS_DISABLE';
     public const CUSTOMERS_ENABLE = 'CUSTOMERS_ENABLE';
     public const CUSTOMERS_UPDATE = 'CUSTOMERS_UPDATE';
+    public const CUSTOMERS_SMS_MARKETING_SUBSCRIBED = 'CUSTOMERS_SMS_MARKETING_SUBSCRIBED';
```

Comment:
@paulomarg Is this file auto-updated?

`CUSTOMERS_SMS_MARKETING_SUBSCRIBED` hasn't shipped yet, it probably shouldn't ship with this change.